package.json

```diff
@@ -11,6 +11,9 @@
 		"type": "git",
 		"url": "https://github.com/dojo/loader.git"
 	},
+	"dependencies": {
+		"dojo-core": "dojo/core"
```

Comment:
There's no reason this needs `dojo-core`. It should be fairly simple and along these lines:

```ts
// This function is needed so onreadystatechange can be set to something
// that doesn't form a closure around the XHR object and form a circular
// reference
const noop = function () {};
getText = function(url: string, callback: (value: string) => void): void {
    const xhr = new XMLHttpRequest();
    xhr.onreadystatechange = function () {
        if (xhr.readyState === 4) {
            xhr.onreadystatechange = noop;
            callback(xhr.responseText);
        }
    };
    xhr.open('GET', url, true);
    xhr.send(null);
};
```